bins/stackablectl/src/cmds/operator.rs

```diff
@@ -72,18 +93,37 @@ Use \"stackablectl operator describe <OPERATOR>\" to get available versions for
     operators: Vec<OperatorSpec>,
 
     /// Type of local cluster to use for testing
-    #[arg(short, long, value_enum, value_name = "CLUSTER_TYPE", default_value_t = ClusterType::default())]
+    #[arg(short = 'c', long = "cluster", value_enum, value_name = "CLUSTER_TYPE", default_value_t = ClusterType::default())]
     #[arg(
         long_help = "If specified, a local Kubernetes cluster consisting of 4 nodes (1 for
 control-plane and 3 workers) will be created for testing purposes. Currently
 'kind' and 'minikube' are supported. Both require a working Docker
 installation on the system."
     )]
-    cluster: ClusterType,
+    cluster_type: ClusterType,
 
     /// Name of the local cluster
     #[arg(long, default_value = DEFAULT_LOCAL_CLUSTER_NAME)]
     cluster_name: String,
+
+    /// Number of total nodes in the local cluster
+    #[arg(long, default_value_t = 3)]
```

Comment:
Currently this describes the **total** number of cluster nodes. The lowest this can go is 2 as the cluster needs one control-plane node and one worker node. But I can change the default value to `2`.

Reply:
Oh sorry, 2 than please

Reply:
It's valid to run a single node as both a control plane and worker (although it's definitely not recommended for production setups).

Reply:
Changed in 5b0c371.